benchmarks/graphics_pipeline/main.cpp

```diff
@@ -528,18 +534,109 @@ void ProjApp::Setup()
             const uint32_t sphereVertexCount = mesh.GetCountPositions();
             const uint32_t sphereTriCount    = mesh.GetCountTriangles();
 
+            PPX_LOG_INFO("LOD: " << lod.name);
+            PPX_LOG_INFO("  Sphere vertex count: " << sphereVertexCount << " | triangle count: " << sphereTriCount);
+
+            // Single-sphere geometries
+            Geometry lowPrecisionInterleavedSingleSphere;
+            PPX_CHECKED_CALL(Geometry::Create(
+                GeometryOptions::InterleavedU32(grfx::FORMAT_R16G16B16_FLOAT)
+                    .AddTexCoord(grfx::FORMAT_R16G16_FLOAT)
+                    .AddNormal(grfx::FORMAT_R8G8B8A8_SNORM)
+                    .AddTangent(grfx::FORMAT_R8G8B8A8_SNORM),
+                &lowPrecisionInterleavedSingleSphere));
+
+            Geometry lowPrecisionPositionPlanarSingleSphere;
+            PPX_CHECKED_CALL(Geometry::Create(
+                GeometryOptions::PositionPlanarU32(grfx::FORMAT_R16G16B16_FLOAT)
+                    .AddTexCoord(grfx::FORMAT_R16G16_FLOAT)
+                    .AddNormal(grfx::FORMAT_R8G8B8A8_SNORM)
+                    .AddTangent(grfx::FORMAT_R8G8B8A8_SNORM),
+                &lowPrecisionPositionPlanarSingleSphere));
+
+            Geometry highPrecisionInterleavedSingleSphere;
+            PPX_CHECKED_CALL(Geometry::Create(
+                GeometryOptions::InterleavedU32()
+                    .AddTexCoord()
+                    .AddNormal()
+                    .AddTangent(),
+                &highPrecisionInterleavedSingleSphere));
+
+            Geometry highPrecisionPositionPlanarSingleSphere;
+            PPX_CHECKED_CALL(Geometry::Create(
+                GeometryOptions::PositionPlanarU32()
+                    .AddTexCoord()
+                    .AddNormal()
+                    .AddTangent(),
+                &highPrecisionPositionPlanarSingleSphere));
+
+            // Full geometries
             Geometry lowPrecisionInterleaved;
-            PPX_CHECKED_CALL(Geometry::Create(GeometryOptions::InterleavedU32(grfx::FORMAT_R16G16B16_FLOAT).AddTexCoord(grfx::FORMAT_R16G16_FLOAT).AddNormal(grfx::FORMAT_R8G8B8A8_SNORM).AddTangent(grfx::FORMAT_R8G8B8A8_SNORM), &lowPrecisionInterleaved));
+            PPX_CHECKED_CALL(Geometry::Create(
```

Comment:
Could we change this slightly to improve readability?
For example:
```
Geometry lowPrecisionInterleavedSingleSphere;
Geometry lowPrecisionInterleaved;
{
     GeometryOptions lowPrecisionInterleavedOptions = GeometryOptions::InterleavedU32(grfx::FORMAT_R16G16B16_FLOAT)
                    .AddTexCoord(grfx::FORMAT_R16G16_FLOAT)
                    .AddNormal(grfx::FORMAT_R8G8B8A8_SNORM)
                    .AddTangent(grfx::FORMAT_R8G8B8A8_SNORM);
     PPX_CHECKED_CALL(Geometry::Create(lowPrecisionInterleavedOptions, lowPrecisionInterleavedSingleSphere));
     PPX_CHECKED_CALL(Geometry::Create(lowPrecisionInterleavedOptions, lowPrecisionInterleaved));
}
```
and similarly for others.

Reply:
Thanks, changed